src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/PriceControls.tsx

```diff
@@ -47,6 +55,7 @@ const PriceControls: React.FC<Props> = ({
     currencyPair.pipsPosition,
     currencyPair.ratePrecision,
   )
+  const userError = Boolean(inputValidationMessage)
```

Comment:
I can see you have matched the prop by naming this `userError` which is fine. If we were to refactor this a little including the `<TileBookingSwitch />` component, a nice style is to prefix boolean values with `has`, `is`, or `should` eg `hasUserError` or simply `hasError`

Reply:
good spot, I will change it